internal/impl/devbox.go

```diff
@@ -384,48 +385,32 @@ func (d *Devbox) GenerateEnvrc(force bool, source string) error {
 		)
 	}
 	// confirm .envrc doesn't exist and don't overwrite an existing .envrc
-	if cmdutil.Exists("direnv") {
-		// prompt for direnv allow
-		var result string
-		isInteractiveMode := isatty.IsTerminal(os.Stdin.Fd())
-		if isInteractiveMode {
-			prompt := &survey.Input{
-				Message: "Do you want to enable direnv integration for this devbox project? [y/N]",
-			}
-			err := survey.AskOne(prompt, &result)
-			if err != nil {
-				return errors.WithStack(err)
-			}
+	if source == "generate" {
+		if err := nix.EnsureNixInstalled(
+			d.writer, func() *bool { return lo.ToPtr(false) },
+		); err != nil {
+			return err
 		}
 
-		if strings.ToLower(result) == "y" || !isInteractiveMode || source == "generate" {
-			if err := nix.EnsureNixInstalled(
-				d.writer, func() *bool { return lo.ToPtr(false) },
-			); err != nil {
-				return err
-			}
-
-			// generate all shell files to ensure we can refer to them in the .envrc script
-			if err := d.ensurePackagesAreInstalled(ctx, ensure); err != nil {
-				return err
-			}
+		// generate all shell files to ensure we can refer to them in the .envrc script
+		if err := d.ensurePackagesAreInstalled(ctx, ensure); err != nil {
+			return err
+		}
 
-			// .envrc file creation
-			err := generate.CreateEnvrc(tmplFS, d.projectDir)
-			if err != nil {
-				return errors.WithStack(err)
-			}
+		// .envrc file creation
+		err := generate.CreateEnvrc(tmplFS, d.projectDir)
+		if err != nil {
+			return errors.WithStack(err)
 		}
 
-		if strings.ToLower(result) == "y" || !isInteractiveMode {
+		if cmdutil.Exists("direnv") {
```

Comment:
based on the conversation with @Lagoja, I thought we were gonna print some information messages, like:
```
Success: generated .envrc file
Success: ran `direnv allow`
```

Do you still wanna do that?

Reply:
You could add a `ux.Fsuccess(msg)` method that prints a green "Success:" and the rest of the `msg`

similar to `ux.Fwarning` with the yellow "Warn"

Reply:
yeah sure I'll add that.